inc/entityconfig.class.php

```diff
@@ -303,6 +332,10 @@ static function isNewID($ID) {
       return (($ID < 0) || !strlen($ID));
    }
 
+   /**
+    * Displays form
```

Comment:
FYI: Display form when the item is displayed from an related entity. We might improve the description of the method. Other methods in the plugin have a similar naming. Might be usefun to improve them the same way.